packages/perl.rb

```diff
@@ -3,41 +3,58 @@
 class Perl < Package
   description 'Perl 5 is a highly capable, feature-rich programming language with over 29 years of development.'
   homepage 'https://www.perl.org/'
-  version '5.32.0'
+  @_ver = '5.32.1'
+  version @_ver
   license 'GPL-1+ or Artistic'
   compatibility 'all'
-  source_url 'http://www.cpan.org/src/5.0/perl-5.32.0.tar.gz'
-  source_sha256 'efeb1ce1f10824190ad1cadbcccf6fdb8a5d37007d0100d2d9ae5f2b5900c0b4'
+  source_url "http://www.cpan.org/src/5.0/perl-#{@_ver}.tar.gz"
+  source_sha256 '03b693901cd8ae807231b1787798cf1f2e0b8a56218d07b7da44f784a7caeb2c'
 
-  binary_url ({
+  binary_url({
     aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/perl/5.32.1_armv7l/perl-5.32.1-chromeos-armv7l.tar.xz',
      armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/perl/5.32.1_armv7l/perl-5.32.1-chromeos-armv7l.tar.xz',
        i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/perl/5.32.1_i686/perl-5.32.1-chromeos-i686.tar.xz',
-     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/perl/5.32.1_x86_64/perl-5.32.1-chromeos-x86_64.tar.xz',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/perl/5.32.1_x86_64/perl-5.32.1-chromeos-x86_64.tar.xz'
   })
-  binary_sha256 ({
+  binary_sha256({
     aarch64: '8f96c4ac4e4927ff643670ba362568e5483ba4ef970acd742877ef49b48829df',
      armv7l: '8f96c4ac4e4927ff643670ba362568e5483ba4ef970acd742877ef49b48829df',
        i686: '2193da9719f91be5aa261471aae0314b19da430ae3f09006c30d7d276b5f230d',
-     x86_64: '8c12ba41052f0f49f1dccc7cc1de795d4b4c55ba70c4ed1eff866580405e300a',
+     x86_64: '8c12ba41052f0f49f1dccc7cc1de795d4b4c55ba70c4ed1eff866580405e300a'
```

Comment:
Maybe I'm just tired but shouldn't these binaries be new to this version?

Reply:
They SHOULD be. But it looks like you inadvertently edited in the wrong versions when adjusting for gitlab URLs?

https://github.com/skycocker/chromebrew/blob/f7c9a99f2d7b897ef85b6a17511aa5a6c89252f0/packages/perl.rb